### DISCUSSION TITLE
Final Columns for  prediction Models

### DISCUSSION THREAD

**Discussion by sudipbhattarai820** [ravi7522/Football-Prediction#39] [General]
![Screenshot 2023-08-06 172913](https://github.com/ravi7522/Football-Prediction/assets/105615132/b8611b77-f035-4209-8ba3-1a9ceef6d9ea)
This is for @zeelsheladiya

**Comment by zeelsheladiya** [comment]
thank you so much @sudipbhattarai820. Appreciated

**Comment by ravi7522** [comment]
These are the final list for the column names.

home_team_overall_score
away_team_overall_score
home_total_goal
away_total_goal
home_players_avg_overall_rating
home_players_avg_overall_score
home_players_avg_ideal_body_rate
away_players_avg_overall_rating
away_players_avg_overall_score
away_players_avg_ideal_body_rate
home_total_win
home_total_loose
home_total_draw
away_total_win
away_total_loose
away_total_draw
league_home_total_win
league_home_total_loose
league_home_total_draw
league_away_total_win
league_away_total_loose
league_away_total_draw
